app/templates/sales-orders/show.hbs

```diff
@@ -13,13 +13,15 @@
     toggleOrderState=(route-action "toggleOrderState" model)}}
 
 <div class="row top wrap">
-  {{charts/location-item-info
+  {{#if salesData.length}}
```

Comment:
let's change to `hasSalesData` instead of using length. General style rule, I think it's clearer to create a `gt` length rather than directly referencing. This way if we need to add special filters, we don't need to refactor templates.